wiki/Help_centre/Store/en.md

```diff
@@ -2,124 +2,48 @@
 
-### The LEDs on my osu! nono keyboard aren't working! {id=osu!keyboard-leds}
+Sometimes orders can get stuck in the postal system. If your order is stuck somewhere in the United States, it may be possible to track down through the [USPS](https://faq.usps.com/s/article/USPS-Tracking-The-Basics), usually by contacting the relevant location on your tracking info. If your order is stuck outside of the United States, contact your country's postal service for help.
 
-**This can be due to a few reasons - corrosion between the LEDs and the mainboard, or in certain circumstances with earlier models, faulty LEDs.**
+Feel free to contact [merch@ppy.sh](mailto:merch@ppy.sh) if you're still confused, but note that after a product leaves our hands, we know as much about the status of the delivery as you do.
 
-Please contact [store@ppy.sh](mailto:store@ppy.sh) for further inquiries.
+## I need help with a specific product!
 
-#### Checking the LEDs for corrosion {id=osu!keyboard-corrosion}
+### osu! keycaps {id=osu!-keycaps}
 
-**Rubbing the base of the LED connectors with a small square of aluminium foil will remove most of the residue caused from corrosion.**
+[osu! keycaps](https://www.thekapco.com/products/osu-pippi-keycap-set) are produced and shipped by The KapCo, so we recommend [speaking with their customer support](https://www.thekapco.com/pages/faqs) for any keycap-specific issues.
 
-Corrosive residue usually appears to be blackish-grey, or may appear as strange stains on the metal.
+If you're not able to get an adequate response from them, email [merch@ppy.sh](mailto:merch@ppy.sh) and we'll see what we can do.
 
-Removing this residue may restore your LED to working condition. If it does, you know how to fix it next time!
+### osu! deskmat {id=osu!-deskmat}
 
-## osu!tablet {id=osu-tablet}
+While most osu! products are shipped from our warehouse in the United States, [osu! deskmats](https://osu.ppy.sh/store/products/1347) are shipped from different hubs in the United Kingdom, the United States, and Australia, so we don't have direct visibility over every item that is delivered.
 
-### My osu!tablet has stopped working or doesn't work at all! {id=osu-tablet-not-working}
+If your product has a problem of any sort, contact us at [merch@ppy.sh](mailto:merch@ppy.sh) with your order number and an explanation of the problem.
 
-**This can be tricky to troubleshoot since the osu!tablet is a two-unit solution (the tablet and the pen).**
+## None of this applies to what I need help with! {id=none}
 
-Given that the osu!tablet is a two-unit solution (e.g. tablet and pen), it can be difficult to tell which unit is experiencing issues when things go wrong.
-
-To find out, perform the following steps:
```

Comment:
Kinda sad to lose this historical context 😢

Could we perhaps make an archive / similar section at the bottom of the page? Or is it just noise at this point?

Reply:
i re-added previous pages under `Help_centre/Store_archive`, i dont think it'd be good to crowd this page with stuff that's not really relevant anymore